src/Monolog/Formatter/MongoDBFormatter.php

```diff
@@ -100,6 +102,13 @@ protected function formatException(\Exception $exception, $nestingLevel)
 
     protected function formatDate(\DateTime $value, $nestingLevel)
     {
-        return new \MongoDate($value->getTimestamp());
+        $seconds = (int) $value->format('U');
+        $milliseconds = (int) $value->format('u') / 1000;
+
+        if ($seconds < 0) {
+            return new UTCDateTime($seconds * 1000 - $milliseconds);
+        } else {
+            return new UTCDateTime($seconds * 1000 + $milliseconds);
+        }
```

Comment:
Yeah it was an optional formatter.. I'm not sure if it is still relevant with the latest mongo clients or not. I know nothing about mongo :)